packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -161,7 +161,7 @@ Image.propTypes = {
       alternativeText: PropTypes.string,
       width: PropTypes.number,
       height: PropTypes.number,
-    }).isRequired,
+    }),
```

Comment:
Why is it no longer required?

Reply:
Actually when there is a empty node or text node, and user tries to select image option, that time schema becomes as below, so this required condition violates. I have also added a condition when `!element.image return null` inside <Image /> element.
`
{
type: 'image',
children: [
   { 
      type: 'text', 
      text: ''
   }
]
}
`